pages/history.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from '../styles/MoodHistory.module.css'
+import Footer from '../components/footer'
+
+const MoodHistory = () => {
```

Comment:
This isn't doctrine and you'll find other opinions on this, but I've found that most of the companies that I work with tend to build JS style guide around named exports over default exports. On the export side, it looks something like this:

```
export const History = () => {
...
}
```

On the import side, it changes the syntax a little bit:
```
import { History } from "path/to/History.tsx"
```

The advantage here is that the import name stays consistent between declaration and import with a named import, where a default import can be named anything, which can cause some drama on larger projects. I'm hesitant to call this a best practice, but it probably falls somewhere near that line IMHO.